### ISSUE TITLE
Keymap issues with Alt

### ISSUE THREAD

**Issue by jaredpar** [jaredpar/VsVim#806]
Just one question: How do I map modifier key combinations. E.g. I use these with Vim (and previously in VS with ViEmu)

```
map j <down>zz

map k <up>zz

map <a-j> 4<down>zz

map <a-k> 4<up>zz

map <s-a-j> 16<down>zz

map <s-a-k> 16<up>zz

map <c-a-j> 64<down>zz

map <c-a-k> 64<up>zz
```

This keeps my cursor centered vertically and enables me to scroll up/down by 1, 4, 16, or 64 lines when also holding down one or more modifier keys. My _vsvimrc is correctly read but :map shows different mappings, and the modifier keys have no effect, for instance:

```
map <a-j> 4<down>zz

map <a-k> 4<up>zz

   - displays as:

<M-J> 4<Down>

<M-K> 4<Up>zz
```

**Comment by juliuszint** [comment]
In reference to #2037. The Keystroke gets recognized perfektly fine in the VsVim plugin. But it doesnt get handled because in Input Mode every Keystroke should first be handled by Visual Studio if I understand the comment correctly. 

If i comment out the aformentioned check the keybinding works perfectly fine. I forgot to reference the file where the code snipped is from `VsVimKeyProcessor.cs`. But just commenting out the check is obviously not going to fly.

Do you see a problem in VsVim handling keystrokes in Input Mode that do not output a character?

**Comment by Widgertius** [comment]
Sorry for necro bumping. Was this ever achieved? Trying to get similar behavior as what was on the first post. Thanks.

**Comment by ricksladkey** [comment]
This was fixed in PR #2206 and so this issue can be closed.

**Comment by ricksladkey** [comment]
Note that after any mapping command, you may need to go to "Tools -> Options -VsVim" and select "Handled by VsVim" for a newly conflicting key binding.
